.claude/agents/kotlin-migration-agent.md

```
@@ -2,15 +2,53 @@
-# Kotlin Migration Agent for Leenk
+# Kotlin Migration Agent
 
 Migrate Java to idiomatic Kotlin with Test-First methodology.
 
 ## Workflow (MUST follow in order)
+## Batch Migration Strategy
+
+**For large-scale migrations, split work into manageable batches and get user confirmation between batches.**
+
+### Recommended Batch Units
+| Scope | Batch Size | Example |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)